### DISCUSSION TITLE
How to change the theme of Coding Stats

### DISCUSSION THREAD

**Discussion by geekpro404** [mrhrifat/mrhrifat#3] [Q&A] [answered]
- I want to change the theme color of `Rakibul Hassan Rifat Coding Stats` in my own profile. Is it possible?

**Comment by mrhrifat** [comment, marked as answer]
Yup.. This is possible. The author of this Open Source has created many themes for styling. 
- There is about 10 inbuilt theme like;
```
dark, radical, merko, gruvbox, tokyonight, onedark, cobalt, synthwave, highcontrast, dracula
```


But If you want to add custom theme. This is also possible. Check the `Theme config` file for more.
